### ISSUE TITLE
Show viewers/people on each channel

### ISSUE THREAD

**Issue by LiamDawe** [vinszent/gnome-twitch#253]
One thing I've noticed missing, is the ability to see viewer count and get a list of who is chatting. Any chance of seeing that in it?

**Comment by vinszent** [comment]
Definitely! For v0.5.0 I am planning on implementing a lot of new features and redesigning the UI mainly around the player and chat. This will include exactly what you said: see viewer count, display chat list, follow channel directly, etc in the player and not just in the channel list.

**Comment by ptkato** [comment]
Would be nice if one could open the stream even if it's offline, only for chatting purposes. Right now the chat is kinda stuck within the stream itself.
